internal/cliwrapper/cliwrapper_test.go

```diff
@@ -48,11 +49,59 @@ func TestPodman_ImageExists(t *testing.T) {
 	tests.RemoveImage(logger, tests.TestImage)
 }
 
+func TestPodman_ImageExists(t *testing.T) {
+	Podman_ImageExists(t, "")
+}
+
+func TestPodman_Remote_ImageExists(t *testing.T) {
+	var tmpPodmanSocketCmd *exec.Cmd
+
+	// Check if there is an existing connection of `podman-machine-default` since this is included when installing
+	// podman desktop for macOS.
+	connectionName := "podman-machine-default"
+	chkDefaultConnectionCmd := exec.Command(tests.GetPodmanPath(), "--connection", connectionName, "system", "info") //nolint:gosec
+	if err := chkDefaultConnectionCmd.Run(); err == nil {
+		Podman_ImageExists(t, connectionName)
+	} else if runtime.GOOS == "linux" {
+		// The podman-machine-default doesn't exist then for Linux, create a temporary socket
+
+		// Setup
+		connectionName = "arcaflow-engine-deployer-podman-test"
+		podmanSocketPath := "unix:///var/tmp/" + connectionName + ".sock"
+
+		tmpPodmanSocketCmd = exec.Command(tests.GetPodmanPath(), "system", "service", "--time=0", podmanSocketPath)
```

Comment:
Should we annotate with `//#nosec G204` to disable this annoying `gosec` message?

Reply:
Yup, or with `//nolint:gosec`, but I like your more specific one, better (but existing practice seems to be the bigger hammer).